src/pages/project/components/ProjectIconHeader.tsx

```diff
@@ -5,16 +5,17 @@ import { ProjectStatus } from '../../../types/project.types';
 
 interface ProjectIconHeaderProps {
   projectStatus?: ProjectStatus;
+  fontColor?: string;
 }
 
-export const ProjectIconHeader = ({ projectStatus }: ProjectIconHeaderProps) => {
+export const ProjectIconHeader = ({ projectStatus, fontColor = '' }: ProjectIconHeaderProps) => {
   const { t } = useTranslation();
 
   return (
-    <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', mb: '0.5rem' }}>
+    <Box sx={{ display: 'flex', gap: '0.25rem', alignItems: 'center', mb: '0.5rem' }}>
       <ProjectIcon />
-      <Typography>{t('project.project')}</Typography>
-      <Typography sx={{ fontWeight: 'bold' }}>
+      <Typography sx={{ color: fontColor }}>{t('project.project')}</Typography>
+      <Typography sx={{ fontWeight: 'bold', color: fontColor }}>
```

Comment:
Usikker på hvorfor dette virker, kanskje noe med arv? Jeg sender aldri fargen inn noe sted fordi det ikke trengs. Men gjør jeg det så påvirker det fargen da.

Reply:
Er det ikke bare default-farge som brukes da? 🤔 
Ser ut som det er likt om vi tar vekk alt av `color` her også, så stemmer for å bare fjerne det

Reply:
Med mindre vi vet vi vil ha ulike farger basert på parent her da, men da virker det også mest natulig å legge til når det skal brukes

Reply:
Ah, ser det trengs for landing page ja, mens det virker uansett i wizard! Da skjønner jeg hvorfor det er lagt til, men ikke hvorfor det virker sånn 😅 Skal se litt på det, for det gir ikke mening for meg sånn umiddelbart iaf, så det er en prop jeg typisk ville fjerne når jeg kommer over det

Reply:
aha, tror jeg fant det. Om man bruker `color: 'sdfsd'` så blir det også likt, så tror den får som streng, som ikke er en gyldig CSS-farge, og da defaulter den til `inherit`.
Så det vi vil ha her er `color: "inherit"`, sånn at den bevisst arver fra parent.
Skulle tro at fravær av verdi og feil verdi ga samme utfall da, men tydeligvis ikke..